lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -967,11 +975,12 @@ def auth_was_for_cim?(authorization)
         action && is_cim_action?(action)
       end
 
-      def parse_direct_response_elements(response)
+      def parse_direct_response_elements(response, options)
         params = response[:direct_response]
+        delimiter = options[:delimiter] if options[:delimiter]
         return {} unless params
 
-        parts = params.split(',')
+        parts = params.split(delimiter || ',')
```

Comment:
Because you're ending with `|| ','` here, you can just axe line 980 entirely and turn this into `parts = params.split(options[:delimiter] || ',')`

Reply:
That's what I thought initially but second-guessed myself, restoring.